builder/test/visualize/parallel_stencils_1.html

```diff
@@ -373,15 +373,18 @@
                   let __loop_step = 2;
                   for(let y = __loop_min; y <= __loop_max; y += __loop_step) {
                     { let __intm3 = crop_dim(intm3, 1, {min:y, max:min((y + 1), buffer_max(out, 1))});
-                      let __loop_min = y;
-                      let __loop_max = min((y + 1), buffer_max(out, 1));
-                      let __loop_step = 1;
-                      for(let y = __loop_min; y <= __loop_max; y += __loop_step) {
-                        { let __intm3 = crop_dim(intm3, 1, {min:NaN, max:y});
-                          consume(intm1);
-                          produce(intm3);
-                          __event_t++;
-                          intm3 = __intm3;
+                      {
+                        let y_min_orig0 = max(y, buffer_min(out, 1));
```

Comment:
Here it looks like sliding window did something, but was kind of a no-op? Is it a bug, or did the simplifier fix up something that was real sliding?

Reply:
I dug into this and this seems like something which is happening *after*  slide_and_fold_storage. I compared main vs this branch and the results of slide_and_fold_storage are identical wrt to what is slided or folded. Maybe simplifier does something slightly differently, because of the some of the bounds changes?

Reply:
That's surprising, is your branch based on the current main branch?

Either way, I think it's fine to ignore this, nothing is actually wrong here, just simplification noise I guess.